pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -93,7 +95,8 @@ var _ = Describe("Node-labeller config", func() {
 		err := nlController.loadDomCapabilities()
 		Expect(err).ToNot(HaveOccurred())
 
-		Expect(nlController.loadHostSupportedFeatures()).To(Succeed())
+		err = nlController.loadHostSupportedFeatures()
```

Comment:
Any reasons to change this?

Reply:
To keep it the same with all the other similar calls. For Example the one directly above.

Reply:
Sure thing, i will push it together with whatever change we decide for the if clause